webpack/assets/javascripts/react_app/components/Architectures/Welcome.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl } from '../../common/helpers';
+
+export const WelcomeArchitecture = ({ canCreate }) => {
+  const action = canCreate && {
+    title: __('Create Architecture'),
+    url: foremanUrl('/architectures/new'),
+  };
+  const content = __(`Each entries represent a particular hardware architecture, most commonly <b>x86_64</b> or <b>i386</b>.
+  Foreman also supports the Solaris operating system family, which includes <b>sparc</b> based systems.`);
+  const description = (
+    <>
+      {__(
+        'Before you proceed to using Foreman you should provide information about one or more architectures.'
+      )}
+      <br />
+      {<div dangerouslySetInnerHTML={{ __html: content }} />}
```

Comment:
I don't think we need to insert it dangerously.
can we just wrap that content with <p> or something? or is the `EmptyState` component doing problems?

Reply:
It is needed because the string contains html and we can't break it into partial strings since that will break the translation. The same for the rest of the strings. We came up with this solution since the sentences can't be broken up due to the way translations work.

Reply:
@laviro please read the comment regarding this in my previous review - https://github.com/theforeman/foreman/pull/8418#discussion_r609429894

Reply:
Interesting, I like it, though maybe `sprintf` will work better in this case, though I didn't try it :D
e.g: 
```js
sprintf(__('Each entries represent a particular hardware architecture, most commonly %s)'), 'x86_64'.bold)
```